python/tutorials/06-fused-attention.py

```diff
@@ -199,13 +200,12 @@ def _bwd_kernel(
             q = tl.load(q_ptrs)
             # recompute p = softmax(qk, dim=-1).T
             if CAUSAL:
```

Comment:
I think this if/else is for initializing q. The else case sets it to 0 and then you can pull the tl.dot out since it is common to both.

Reply:
good idea!
I'll do this in separate PR, maybe it worth to rework this part a little to make it easier to merge with upstream: https://github.com/openai/triton/blob/main/python/tutorials/06-fused-attention.py#L198-L201

Reply:
PR #328